### ISSUE TITLE
Error: No JSON at Incomin...

### ISSUE THREAD

**Issue by nappyjim** [586837r/node-red-contrib-alexa-remote2#183]
Get this randomly.  See attached picture, i ended up putting debug on each node (the set volume, the template, and the speak).

Ran it through inject 10x in a row, happened 1/10 times.  Nothing different between that 1 time it errored.

![image](https://user-images.githubusercontent.com/71914301/102954923-4cdaeb80-449a-11eb-8b4e-29f58dc970b7.png)

**Comment by nappyjim** [comment]
I heard this may be because of 2 commands too quick.  I removed the separate volume and speak nodes, and combined them into 1 node via the serial dropdown option.  Will see if that works.

**Comment by andesse** [comment]
Tried a lot around with these issue and I solved it for me. 
I can confirm that happens when too many request appear in short time. Using 3-4 second delays between the request helped me. For some time dependent request I used IFTTT>Alexa what helps too. The delay is like 500ms higher. 
I figured out that using the smarthome node can handle a lot more requests without having an issue (2-3 at once doesn't cause a problem for me) it mostly appears on the echo and routine node.
